### DISCUSSION TITLE
Search by focal length

### DISCUSSION THREAD

**Discussion by laterwet** [photoprism/photoprism#1936] [Ideas]
I tried many things to search by focal length without success. I checked the [search filter documentation](https://docs.photoprism.app/user-guide/organize/search/#search-filter-overview) and the [search source code](https://github.com/photoprism/photoprism/blob/2636255c952267314d4caa268fad104651baa8e0/internal/form/search_photos.go), and can't find anything related to it.

Similar to `fmin` and `fmax` to narrow aperture, I think it would be nice to narrow focal length (either real or 35mm equivalent).

Also, [Lightroom has a feature](https://petapixel.com/2016/12/30/quick-tip-use-lightroom-figure-next-prime-lens/) to show statistics about photos, so you can see which camera/lens/focal length are used most.

What do you think?

**Comment by lastzero** [comment]
Main impediment is that there is no index on these fields so a search *could* be slow if we implement it.
